src/main/java/com/cloudbees/simplediskusage/QuickDiskUsage.java

```diff
@@ -152,6 +155,12 @@ public void run() {
 
                     Thread.sleep(1000); //To keep load average nice and low
                 }
+                File tmpDir = new File(System.getProperty("java.io.tmpdir"));
+                usage.put(new DiskItem(
+                                "java.io.tmpdir",
```

Comment:
I don;t think a user will always know what this is - why not call it "System temporary files"